index.html

```diff
@@ -12,39 +12,175 @@
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+        <div class="box">
+            <img src="req/tonic_2.png" alt="multi" class="banner">
+            <h3>Tonic</h3>
+                <ul>
+                    <li class="dot1">CANOPY</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot2">Back End Dev</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot3">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+        <div class="box">
+            <img src="req/multi_2.png" alt="multi" class="banner">
+            <h3>Multi-Post
+                Stories</h3>
+                <ul>
+                    <li class="dot1">CANOPY</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot2">Back End Dev</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot3">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+     </div>
+    </section>
+    <section>
+        <div class="about">
+            <h1 class="about-header">About<br>Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="supportingText">LET’S CONNECT</p>
+            <ul class="icon">
+                <li>
+                    <a href="#"><img src="req/twitter.png" alt="twitter"></a>
+                </li>
+                <li>
+                <a href="#"><img src="req/linkedin.png" alt="linkedin"></a>
+                </li>
+                <li>
+                <a href="#"><img src="req/M.png" alt="M"></a>
+                </li>
+                <li>
+                <a href="#"><img src="req/github.png" alt="github"></a>
+                </li>
+                <li>
+                <a href="#"><img src="req/V.png" alt="V"></a>
```

Comment:
- [ ] [OPTIONAL] If you consider allowing visitors to connect or contact you - which I believe that is why you added the media, please consider inserting your profile link or username on those media platforms in the `href` instead of the `#` that you currently use. This is good for good user experience.